hugo/layouts/index.html

```diff
@@ -27,9 +27,7 @@
         {{ partial "home/blog.html" . }}
     {{ end }}
 
-    {{ partial "footer/slack.html" . }}
-    {{ partial "footer/contact-us.html" . }}
-    {{ partial "footer/sitemap.html" . }}
+    {{ partial "footer.html" . }}
```

Comment:
Do we know for sure not all footer components are going to be in all pages? Because maybe in the new design they do. Anyway, I prefer having a parametric `footer.html` whose `contact-us` can be hidden.

The slack form I'm gonna separate it because that's probably only going to be used in the home page.